packages/schemas/src/seeds/application.ts

```diff
@@ -35,3 +39,33 @@ export const createDefaultAdminConsoleApplication = (): Readonly<CreateApplicati
     type: ApplicationType.SPA,
     oidcClientMetadata: { redirectUris: [], postLogoutRedirectUris: [] },
   });
+
+export const createTenantMachineToMachineApplication = (
+  tenantId: string
+): Readonly<CreateApplication> =>
+  Object.freeze({
+    tenantId: adminTenantId,
+    id: generateStandardId(),
+    name: 'Cloud Service',
+    description: `Machine to machine application for tenant ${tenantId}`,
+    secret: generateStandardId(),
+    type: ApplicationType.MachineToMachine,
+    oidcClientMetadata: {
+      redirectUris: [],
+      postLogoutRedirectUris: [],
+    },
+    customClientMetadata: {
+      tenantId,
+    },
+  });
+
+export const createAdminTenantApplicationRole = (
```

Comment:
will it be better to add some comments for this? since the name is ambiguous (it also means "to create a role of application"). you can use the comment format below to let [IntelliSense](https://code.visualstudio.com/docs/editor/intellisense) recognizes

```ts
/** ... */
```

Reply:
Is it more about application - role mapping?